daprdocs/content/en/reference/cli/dapr-init.md

```diff
@@ -5,67 +5,70 @@ linkTitle: "init"
+| `--timeout`          |                      | `300`         | The wait timeout for the Kubernetes installation                                     |
+| `--namespace`, `-n`  |                      | `dapr-system` | The Kubernetes namespace to install Dapr in                                          |
+| `--runtime-version`  |                      | `latest`      | The version of the Dapr runtime to install, for example: `1.0.0`                     |
+| `--slim`, `-s`       |                      | `false`       | Exclude placement service, Redis and Zipkin containers from self-hosted installation |
 
-| Name | Environment Variable | Default | Description
-| --- | --- | --- | --- |
-| `--enable-ha` | | `false` | Enable high availability (HA) mode |
-| `--enable-mtls` | | `true` | Enable mTLS in your cluster |
-| `--help`, `-h` | | | Print this help message |
-| `--kubernetes`, `-k` | | `false` | Deploy Dapr to a Kubernetes cluster |
-| `--wait` | | `false` | Wait for Kubernetes initialization to complete |
-| `--timeout` | | `300` | The wait timeout for the Kubernetes installation |
-| `--namespace`, `-n` | | `dapr-system` | The Kubernetes namespace to install Dapr in |
-| `--network` | `DAPR_NETWORK` | | The Docker network on which to deploy the Dapr runtime |
-| `--runtime-version` | | `latest` | The version of the Dapr runtime to install, for example: `1.0.0` |
-| `--slim`, `-s` | | `false` | Exclude placement service, Redis and Zipkin containers from self-hosted installation |
+### Examples
 
-## Examples
+#### Self-hosted environment
 
-### Initialize Dapr in self-hosted mode
 ```bash
 dapr init
 ```
 
-### Initialize Dapr in Kubernetes
+You can also specify a specific runtime version. Be default, the latest version is used.
+
 ```bash
-dapr init -k
+dapr init --runtime-version 0.10.0
 ```
 
-### Initialize Dapr in Kubernetes and wait for the installation to complete
+Dapr can also run [Slim self-hosted mode]({{< ref self-hosted-no-docker.md >}}) without Docker.
 
- You can wait for the installation to complete its deployment with the `--wait` flag.
-
- The default timeout is 300s (5 min), but can be customized with the `--timeout` flag.
 ```bash
-dapr init -k --wait --timeout 600
+dapr init -s
 ```
 
-### Initialize specified version of Dapr runtime in self-hosted mode
+#### Kubernetes environment
+
 ```bash
-dapr init --runtime-version 0.10.0
+dapr init -k
 ```
 
-### Initialize specified version of Dapr runtime in Kubernetes
+You can wait for the installation to complete its deployment with the `--wait` flag.
+The default timeout is 300s (5 min), but can be customized with the `--timeout` flag.
+
 ```bash
-dapr init -k --runtime-version 0.10.0
+dapr init -k --wait --timeout 600
 ```
 
-### Initialize Dapr in [slim self-hosted mode]({{< ref self-hosted-no-docker.md >}})
+You can also specify a specific runtime version.
+
 ```bash
-dapr init -s
+dapr init -k --runtime-version 0.10.0
```

Comment:
Whilst you are updating this file, it would be good to have a non-preview example here. Can you update this to
dapr init -k --runtime-version 1.4.0

Reply:
I fixed it